_component_design/toast.md

```diff
@@ -5,40 +5,136 @@ layout: component
+{% figure [caption:"The toast notification component"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/notifications/toasts/toast-hero.svg)
 {% endfigure %}
 {% endcolumn %}
-{% column left:"hxCol" %}
-{% figure [caption:"Alert"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/toast-alert.svg){:width="100%"}
+
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+### Best Practices
+
+- Keep messages short and succinct, but also helpful. In the case of an error, if there is an action the user can take (correct the errors on the form or submit the form again), use a call to action (CTA) if possible. 
+- Use complete sentences. If only 1 sentence is used, do not include a period. However, if there are 2+ sentences, use periods.
+- Try not to exceed more than 4 lines of your message (excluding the CTA).
+- Only use one CTA per toast. Never include “Dismiss” or “Close” as a CTA. 
+- Clicking on the “X” closes the toast notification. Hovering over it keeps it open until the cursor moves off of it. (See “Timing” section for more detail.)
+{% endcolumn %}
+
+</div>
+
+## Specifications
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Default layout
+
+There are three types of toast notifications: Success/ Confirmation, Error, and Informational.
+
+Clicking on the “X” closes the toast, but it will go away after approx. 10 seconds if no action is taken. (See “Timing” section below for details.) 
+
+While height of toasts are dependent on the amount of text, the minimum height is 56px without a CTA. If a CTA or more lines are needed, the height can increase, but try to keep notification messages short and succinct, and up to 4 lines max. Use complete sentences as it leaves less room for ambiguity. 
+
+Icons are always centered vertically. 
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Default toast notifications specifications"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/notifications/toasts/toast-specifications.svg)
 {% endfigure %}
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Action"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/toast-alert-action.svg){:width="100%"}
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Timing
+
+The toast fades into place over 400ms, sits in place for eight seconds, and fades out over 400ms. If the user happens to hover over the toast while it is fading out, it reanimates immediately (0.0ms) back to full opacity. Likewise, if the user is hovering over the toast at any point, it persists. When the mouse is removed from the expired toast, the toast persists for three more seconds then fades over the normal 400ms. 
```

Comment:
consolidate this entire timing section into the Behaviors section at the bottom of the article, and turn this paragraph into a bullet list to help make the instructions more consumable.